.Gruntfile.babel.js

```diff
@@ -188,6 +190,29 @@ module.exports = (grunt) => {
     grunt.task.run(['exec:eslint', 'exec:puglint', 'exec:stylelint', 'copy', 'sass', rollup])
   })
 
+  grunt.registerTask('cypress', function () {
+    const cypress = require('cypress')
+    const done = this.async()
+    const command = this.flags.open ? 'open' : 'run'
+    const headed = this.flags.headed
+
+    grunt.log.writeln(`cypress: running on "${command}" mode...`)
+
+    cypress[command]({
+      headed
+    })
+      .then((results) => {
+        const code = results.totalFailed ? 1 : 0
+        process.exit(code)
+        return done()
+      })
+      .catch((err) => {
+        grunt.log.writeln('Ups, Cypress did not run!', err)
+        process.exit(1)
+        return done()
```

Comment:
The `done` function needs to be passed something here (instead of your calling `process.exit()` here and above)

```
Either `false` or an `Error` object may be passed to the done function to instruct Grunt that the task has failed.
```

From: https://gruntjs.com/api/inside-tasks

Reply:
Fixed